btr-web/btr-main-app/components/individual-person/AddNew.vue

```diff
@@ -242,7 +242,8 @@ const defaultSI = {
     sharesVotes: {
       registeredOwner: false,
       beneficialOwner: false,
-      indirectControl: false
+      indirectControl: false,
+      inConcertControl: true
```

Comment:
do we want this one to be true by default ?

Reply:
Nope, nice catch. Just put that in to test the text showing up